build-web/build.sh

```diff
@@ -23,6 +23,7 @@ declare -a JOBS_NOTHREADS=(
 export SCONS="scons -j$(expr ${NUM_CORES} / ${NUM_JOBS}) verbose=yes warnings=no progress=no redirect_build_objects=no"
 export OPTIONS="production=yes"
 export OPTIONS_MONO="module_mono_enabled=yes -j${NUM_CORES}"
+export OPTIONS_MONO="module_dotnet_enabled=yes -j${NUM_CORES}"
```

Comment:
```suggestion
export OPTIONS_DOTNET="module_dotnet_enabled=yes -j${NUM_CORES}"
```

There's a typo here, although it's not too important since we don't support web yet. Also, it's interesting that the web script is the only one using `-j` in the options, is that intentional?